redmail/email/body.py

```diff
@@ -63,6 +66,12 @@ def render_table(self, tbl, extra=None):
             raise ImportError("Missing package 'pandas'. Prettifying tables requires Pandas.")
         
         extra = {} if extra is None else extra
+
+        # Allow for pandas styler object, convert to inline CSS for email client rendering
```

Comment:
I believed this also takes care of the "TODO: Nicer tables." in the function, but I will leave that to your discretion to remove.